schematic/schemas/data_model_jsonld.py

```diff
@@ -105,10 +139,12 @@ def create_object(self, template:dict, node:str)->dict:
                 
                 # Get recorded info for current node, and the attribute type
                 node_info = nx.get_node_attributes(self.graph, node_label)[node]
-                
-                # Add this information to the template
-                template[rel_vals['jsonld_key']] =  node_info
-        
+                try:
+                    # Add this information to the template
+                    template[rel_vals['jsonld_key']] =  node_info
+                except:
+                    breakpoint()
```

Comment:
is `breakpoint()` here intended?

Reply:
@linglp this is a working PR currently. I'll be pushing into the main refactor branch as a response to the reviews later when its ready.